presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorMetadata.java

```diff
@@ -71,6 +75,7 @@
 import static com.facebook.presto.spi.type.DateType.DATE;
 import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
 import static com.google.common.base.Ticker.systemTicker;
+import static com.google.common.primitives.Longs.toByteArray;
```

Comment:
Don't static import. Which method is called (the `Ints` vs `Longs` version) is important because you get a different size byte array from each one (so if you see `int x; toByteArray(x)` you don't know which one is being called)